app/views/comments/edit.js.erb

```diff
@@ -0,0 +1 @@
+$("#<%= dom_id(@comment) %>").replaceWith("<%= j(render "comments/form", comment: @comment) %>");
```

Comment:
great job on your comments ray! You can also use variables to make your code more readable.